5.4.0 Regression: Webpack Compilation Error when browserslist is present in project root

<!-- Is this a question? Questions WILL BE CLOSED. Ask in our chat https://on.cypress.io/chat  -->

### Current behavior
<!-- A description including screenshots, stack traces, DEBUG logs, etc -->
I get the error when trying to run tests through the Runner:

```
Error: Webpack Compilation Error
./cypress/integration/main/login-page.test.js
Module build failed (from C:/Users/user/AppData/Local/Cypress/Cache/5.4.0/Cypress/resources/app/packages/server/node_modules/babel-loader/lib/index.js):
C:\Users\user\IdeaProjects\admin\project\project-ui\browserslist:1
# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
^

SyntaxError: Invalid or unexpected token

```

### Desired behavior
<!-- A clear description of what you want to happen -->
The error does not occur.

### Test code to reproduce
<!-- If we cannot fully run the tests as provided the issue WILL BE CLOSED -->
<!-- Issues without a reproducible example WILL BE CLOSED -->

<!-- You can fork https://github.com/cypress-io/cypress-test-tiny repo, set up a failing test, then link to your fork -->

Cypress update to version 5.4.0

Additionally, what I did to fix the problem:

* I deleted the cache from the location: C:\Users\user\AppData\Local\Cypress\Cache
* Deleted from the .node_modules directory the directory from cypress and babel-loader.
* I reinstalled cypress@5.4.0

### Versions
<!-- Cypress, operating system, browser -->
Windows 10 64bit
Browser: Chrome 86, FF 81, Electron 85, Edge86

> Confirmed gerardl123's workaround works:
> 
> ```shell
> git mv browserlists ./browserlistsrc
> ```

I cannot apply this solution.
Because if I implement them, the application is not building during the start.
An unhandled exception occurred: Cannot find module 'browserslist'

--- a/npm/webpack-batteries-included-preprocessor/index.js
+++ b/npm/webpack-batteries-included-preprocessor/index.js
@@ -13,10 +13,11 @@ const getDefaultWebpackOptions = (file, options = {}) => {
       rules: [{
         test: /\.mjs$/,
         include: /node_modules/,
+        exclude: [/browserslist/],
         type: 'javascript/auto',
       }, {
         test: /(\.jsx?|\.mjs)$/,
-        exclude: [/node_modules/],
+        exclude: [/node_modules/, /browserslist/],
         type: 'javascript/auto',
         use: [{
           loader: require.resolve('babel-loader'),
@@ -39,7 +40,7 @@ const getDefaultWebpackOptions = (file, options = {}) => {
         }],
       }, {
         test: /\.coffee$/,
-        exclude: [/node_modules/],
+        exclude: [/node_modules/, /browserslist/],
         loader: require.resolve('coffee-loader'),
       }],
     },

